tests/queries/0_stateless/01090_zookeeper_mutations_and_insert_quorum_long.sql

```diff
@@ -4,7 +4,8 @@ DROP TABLE IF EXISTS mutations_and_quorum2;
 CREATE TABLE mutations_and_quorum1 (`server_date` Date, `something` String) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{database}/test_01090/mutations_and_quorum', '1') PARTITION BY toYYYYMM(server_date) ORDER BY (server_date, something);
 CREATE TABLE mutations_and_quorum2 (`server_date` Date, `something` String) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{database}/test_01090/mutations_and_quorum', '2') PARTITION BY toYYYYMM(server_date) ORDER BY (server_date, something);
 
-SET insert_quorum=2, insert_quorum_parallel=0, insert_quorum_timeout=3000000;
+-- Should not be larger then 600e6 (default timeout in clickhouse-test)
+SET insert_quorum=2, insert_quorum_parallel=0, insert_quorum_timeout=300e3;
```

Comment:
@azat there is issue with test for address sanitizer.

Reply:
Hm, failure is interesting:

```
2021.09.03 05:01:46.411671 [ 712 ] {722b020e-6dea-440b-bdc7-8931264141bf} <Debug> DatabaseCatalog: Waiting for table 130769d4-e6e4-4dd8-9307-69d4e6e49dd8 to be finally dropped
...
2021.09.03 05:11:33.198727 [ 505 ] {} <Information> DatabaseCatalog: Have 1 tables in drop queue (0 of them are in use), will try drop test_hv367u.mutations_and_quorum2 (130769d4-e6e4-4dd8-9307-69d4e6e49dd8)
...
2021.09.03 05:11:35.363742 [ 712 ] {722b020e-6dea-440b-bdc7-8931264141bf} <Debug> MemoryTracker: Peak memory usage (for query): 0.00 B.
2021.09.03 05:11:35.364116 [ 505 ] {} <Trace> test_hv367u.mutations_and_quorum2 (ReplicatedMergeTreeRestartingThread): Threads finished
2021.09.03 05:11:35.364590 [ 712 ] {} <Debug> TCPHandler: Processed in 588.995634033 sec.
2021.09.03 05:11:35.365265 [ 712 ] {} <Debug> TCPHandler: Done processing connection.
```

And all this time server was running other tests, so it seems that waiting of table takes too long.
I guess 712 thread was waiting for the mutex and got it last, although I can't verify this since this is ASAN and trace-log is empty.